doc/content/getting-started/upgrading/_index.md

```diff
@@ -9,58 +9,195 @@ Follow these instructions to upgrade an existing {{% tts %}} instance.
 
 <!--more-->
 
-Replace the tag in your **docker-compose.yml** file with the newer version of {{% tts %}}:
+{{< note >}} All commands explained here are executed in a terminal prompt in the same **directory** as your `docker-compose.yml` file. {{</ note >}}
+
+### Pulling new images
+
+Replace the tag in your `docker-compose.yml` file with the newer version of {{% tts %}}.
+
+{{< tabs/container "Enterprise" "Open Source" >}}
+{{< tabs/tab "Enterprise" >}}
 
 ```yaml
 # file: docker-compose.yml
 services:
   # ...
   stack:
-    image: 'thethingsnetwork/lorawan-stack:<the tag>'
+    image: 'thethingsindustries/lorawan-stack:<new-version-tag>'
   # ...
+```
+{{< /tabs/tab >}}
+{{< tabs/tab "Open Source" >}}
 
+```yaml
+# file: docker-compose.yml
+services:
+  # ...
+  stack:
+    image: 'thethingsnetwork/lorawan-stack:<new-version-tag>'
+  # ...
 ```
 
-Pull the new images:
+{{< /tabs/tab >}}
+{{< /tabs/container >}}
+
+Pull the docker images of the new version.
 
 ```bash
-$ docker-compose pull
-Pulling cockroach ... done
-Pulling redis     ... done
+$ docker-compose pull stack
 Pulling stack     ... done
 ```
 
-Shut down the stack:
+The new version will be pulled locally but not run yet.
+
+### Backup
+
+{{< warning >}} We do not recommend manually backing up and restoring databases for production deployments. Consider using a managed database service with support for snapshots. {{</ warning >}}
+
+Stop `{{< tts >}}` container.
+
+```bash
+$ docker-compose stop stack
+Stopping <directory>_stack_1 ... done
+```
+
+#### Redis
+
+Navigate to the Redis data folder.
+
+This can be found in the `docker-compose.yml` for the `redis` service (ex: `${DEV_DATA_DIR:-.env/data}/redis`)
```

Comment:
```suggestion
Navigate to the Redis data folder.

This can be found in the `docker-compose.yml` for the `redis` service (ex: `${DEV_DATA_DIR:-.env/data}/redis`).
```

Reply:
What can be found, the Redis data folder?